Source/Core/CMakeLists.txt

```diff
@@ -2,7 +2,10 @@ add_subdirectory(AudioCommon)
 add_subdirectory(Common)
 add_subdirectory(Core)
 add_subdirectory(DiscIO)
-add_subdirectory(DolphinWX)
+if(NOT DISABLE_WX)
+  add_subdirectory(DolphinWX)
```

Comment:
@ligfx You are right, the wx cmake won't compile anything (except the old MainNoGUI), but I think this is clearer.